devtools/pip-tools/pr_tool/main.py

```diff
@@ -682,6 +652,40 @@ def main():
         return 2
     logging.info("Using git remote '{}'".format(git_remote))
 
+    if args.subcommand == "backport":
```

Comment:
if the return codes have some deeper meaning, we should probably give them a name. Im not sure i can tell at a glance what the difference between 2 and 1 is (at first i thought git error = 1, setup error = 2 but cherry_pick returns 2 on failure).

Reply:
I guess it is mostly 1 -> something went wrong and 2 -> you (the user) did something wrong.
However, this isn't as clear as it was before. Maybe we should just `return 1` in all the error-cases.

Reply:
maybe we should just return the boolean we get from the function here, this would also make the code nicer.